Aqovia.Utilities.SagaMachine.Tests/StateMachineTests.cs

```diff
@@ -92,33 +91,23 @@ public StateMachineTests()
         [Fact(DisplayName = "Should not allow InitialiseState without Id")]
         public void SagaMachineShouldHaveInitialiseStateId()
         {
-            //Arrange
-            Guid thisId = Guid.NewGuid();
-
             _sagaMachine
                 .WithMessage<HelloMessage>((proccess, msg) => proccess
-                    .InitialiseState((hello) => new TestState
+                    .InitialiseState(hello => new TestState
                     {
                         SomeStateVariable = hello.Message
                     })
                     .Execute()
                 );
 
-            //Act
-
-            //Assert
-            Action shouldThrow = () =>
+            Func<Task> act = async () =>
             {
-
-                _sagaMachine.Handle(new HelloMessage
+                await _sagaMachine.Handle(new HelloMessage
                 {
                     Message = "Initialised"
-                }).Wait();
+                }).ConfigureAwait(false);
             };
-            shouldThrow.ShouldThrow<SagaException>();
-
-
-
+            act.ShouldThrow<SagaException>();
```

Comment:
Well, not trying to be nasty, as I already said this is cosmetic, but writing the very extension and keeping it in your own code for now can be the first step to proposing it to FluentAssertions.

At least your userspace code, in all the places this is the issue will already be doing things properly.
Anyways, this shouldn't be blocking the release, so really Looks More Than Good Enough for me.